src/Resources/views/Invoice/invoice.html.twig

```diff
@@ -0,0 +1,120 @@
+{% set shopBillingData = invoice.shopBillingData %}
+
+<div class="invoice-box">
+    <table cellpadding="0" cellspacing="0">
+        <tr class="top">
+            <td colspan="7">
+                <table>
+                    <tr>
+                        <td colspan="3" class="title">
+                            {% if parrentTemplateType == 'pdf' %}
```

Comment:
1. `parrentTemplateType` -> `parentTemplateType` 
2. In fact, I would change it to `logoIncluded` as the type of the template is unnecessary technical detail